xline-client/src/lib.rs

```diff
@@ -1,4 +1,105 @@
-//! Xline-client
```

Comment:
Could we use `#![doc = include_str!("../README.md")]` here ?
FYI: https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html

Reply:
done